Remove files generated by FE builds

msabramo opened the pull request:
These probably shouldn't be necessary and are bloating the repo and making it harder to work with the code.

## Type

<!-- Select the type of Pull Request -->
<!-- Keep only the necessary ones -->

🧹 Refactoring

msabramo committed:
Remove files generated by FE builds

These probably shouldn't be necessary and are bloating the repo and
making it harder to work with the code.

msabramo commented:
Fixed merge conflict. Consider merging soon if possible to minimize merge conflicts.

krrishdholakia commented:
<img width="386" alt="Screenshot 2025-04-17 at 6 26 06 PM" src="https://github.com/user-attachments/assets/f3a1fa30-104d-4cb7-93b9-4f535541aaed" />

Hey @msabramo the proxy uses this folder

msabramo commented:
Can’t you build the FE files in your CI?

Having them in git is very strange and it causes annoyances when working with the code because they easily become modified and then you can’t switch branches without first throwing them away. Much easier to not have them in git.

krrishdholakia commented:
Yes - we should. 

Since this PR is blocked on that change, we can either close this out for now or refactor this to incorporate the change.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/BerriAI/litellm?pullRequest=10067) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/BerriAI/litellm?pullRequest=10067) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/BerriAI/litellm?pullRequest=10067) it.</sub>

msabramo commented:
I am so tired of seeing this message whenever I am working locally: 😢 

<img width="940" alt="Screenshot 2025-05-06 at 4 33 54 PM" src="https://github.com/user-attachments/assets/cbe0c89a-e6fa-4bb4-87b3-5d90a2837078" />

msabramo commented:
> Yes - we should.
> 
> Since this PR is blocked on that change, we can either close this out for now or refactor this to incorporate the change.

Taking a stab at seeing if I can get the `Dockerfile` to build these files, so that they can be removed...

msabramo committed:
Build admin UI and copy into Docker image

```dockerfile
COPY --from=builder /app/litellm/proxy/_experimental/out /app/ui/litellm-dashboard/out
```

msabramo commented:
> > Yes - we should.
> > Since this PR is blocked on that change, we can either close this out for now or refactor this to incorporate the change.
> 
> Taking a stab at seeing if I can get the `Dockerfile` to build these files, so that they can be removed...

e454d00f1 is my attempt at doing the build and copying it into the Docker image.

msabramo committed:
Merge branch 'main' into remove-generated-files
